devices/MS5611/Sampling.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Iot.Device.MS5611
+{
+    /// <summary>
+    /// MS5611 sampling
+    /// </summary>
+    public enum Sampling : byte
+    {
+        /// <summary>
+        /// 256 ratio
+        /// </summary>
+        UltraLowPower = 0x00,
```

Comment:
Also, like for the constant values, I would encourage you to use the command register as well for the values to make it easier to read and to maintain. Not having full register and playing with adding numbers is always a source of problems.